database/migrations/2018_10_27_230346_fix_non_english_tab_slugs.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Migrations\Migration;
+
+class FixNonEnglishTabSlugs extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::table('tags')->update(['name_slug' => DB::raw('`name`')]);
```

Comment:
This updates every tags->name_slug with the name of tag.
I don't think it's what is intended?

You should do:
```suggestion
        Tag::chunk(200, function ($tags) {
            foreach ($tags as $tag) {
                if (empty($tag->name_slug)) {
                    $tag->forceFill([
                        'name_slug' => htmlentities($tag->name),
                    ])->save();
                }
            }
        });
```
Don't forget to add the use: `use App\Models\Contact\Tag;`

Reply:
Good catch! I've updated now